proto/poktroll/migration/tx.proto

```diff
@@ -143,23 +146,47 @@ message MsgClaimMorseApplication {
 // It indicates the morse_src_address of the account which was claimed, the unstaked
 // balance claimed, the application stake, and the height at which the claim was committed.
 message MsgClaimMorseApplicationResponse {
+  
   // The hex-encoded address of the Morse account whose balance will be claimed.
   string morse_src_address = 1 [(gogoproto.jsontag) = "morse_src_address"];
-
+  
   // The unstaked balance which was claimed.
   cosmos.base.v1beta1.Coin claimed_balance = 2 [(gogoproto.jsontag) = "claimed_balance", (gogoproto.nullable) = false];
-
+  
   // The stake of the application which was staked as a result of the claim.
   // If the application was already staked, this amount does not include the initial stake (i.e. only the portion which was "claimed").
   cosmos.base.v1beta1.Coin claimedApplicationStake = 3 [(gogoproto.jsontag) = "claimed_application_stake", (gogoproto.nullable) = false];
-
+  
   // The height (on Shannon) at which the claim was created.
   int64 claimed_at_height = 4 [(gogoproto.jsontag) = "claimed_at_height"];
-
+  
   // The service ID of the service which the application is staked for.
   string service_id = 5 [(gogoproto.jsontag) = "service_id"];
 
   // The application which was staked as a result of the claim.
   application.Application application = 6 [(gogoproto.jsontag) = "application"];
 }
 
+// TODO_UPNEXT(@bryanchriswhite, #1034): for messages below:
+// - Update field types where necessary.
+// - Rename fields to conform to best practices.
+// - Add applicable field options.
+// - Add comments to fields & messages.
+
+message MsgClaimMorseSupplier {
+  option (cosmos.msg.v1.signer) = "shannonDestAddress";
+  string                   shannonDestAddress = 1;
+  string                   morseSrcAddress    = 2;
+  string                   morseSignature     = 3;
+  cosmos.base.v1beta1.Coin stake              = 4 [(gogoproto.nullable) = false];
+  string                   serviceConfig      = 5;
+}
+
+message MsgClaimMorseSupplierResponse {
+  string                   morseSrcAddress      = 1;
+  cosmos.base.v1beta1.Coin claimedBalance       = 2 [(gogoproto.nullable) = false];
+  string                   serviceId            = 3;
+  cosmos.base.v1beta1.Coin claimedSupplierStake = 4 [(gogoproto.nullable) = false];
+  int32                    claimedAtHeight      = 5;
```

Comment:
Should we add `  poktroll.shared.Supplier supplier = 1;` here?

Reply:
Yes, thank you!